packages/sit-onyx/src/components/OnyxInput/OnyxInput.vue

```diff
@@ -89,6 +89,7 @@ const shouldShowCounter = computed(() => props.withCounter && props.maxlength);
           :minlength="props.minlength"
           :maxlength="props.maxlength"
           :aria-label="props.hideLabel ? props.label : undefined"
+          :title="props.hideLabel ? props.label : undefined"
```

Comment:
could you maybe do a mini-adjustment to align the storybook names of the "without/hidden label" examples?
![image](https://github.com/SchwarzIT/onyx/assets/151019977/60bccb13-980c-43b2-a2c1-027dda6a9a5d)

Reply:
done